src/Compilers/Core/Portable/Operations/ICollectionElementInitializerOperation.cs

```diff
@@ -28,11 +27,16 @@ public interface ICollectionElementInitializerOperation : IOperation
         /// <summary>
         /// Arguments passed to add method invocation.
         /// </summary>
-        ImmutableArray<IOperation> Arguments { get; }
+        ImmutableArray<IArgumentOperation> Arguments { get; }
```

Comment:
>ImmutableArray<IArgumentOperation> Arguments { get; } [](start = 8, length = 53)

Should we also add ```bool IsVirtual { get; }``` property?

Reply:
What would be the value in the IsVirtual property? 

---
In reply to: [187751117](https://github.com/dotnet/roslyn/pull/26781#discussion_r187751117) [](ancestors = 187751117)

Reply:
> What would be the value in the IsVirtual property? 

The same value as in IInvocationOperation.IsVirtual property - makes it clear whether the call is virtual or not. 

---
In reply to: [188026663](https://github.com/dotnet/roslyn/pull/26781#discussion_r188026663) [](ancestors = 188026663,187751117)